draft-patton-cfrg-vdaf.md

```diff
@@ -829,6 +832,22 @@ def vec_add(left, right):
 ~~~
 {: #field-helper-functions title="Common functions for finite fields."}
 
+### FFT-Friendly Fields {#field-fft-friendly}
+
+Some VDAFs require fields that are suitable for efficient computation of the
```

Comment:
As noted here, this is required for the generic FLP construction, in particular because it uses the group generator to pick the points used to construct the gadget polynomials. It would be possible to pick these points differently, but that would be a different construction.

Another option is to make these fixed points a parameter. I see no reason to do this, however.